tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -554,26 +554,26 @@ static void node_configure_and_reset(void)
 					  &status));
 	ASSERT_EQUAL(0, status);
 
-	k_sleep(K_SECONDS(1));
+	k_sleep(K_SECONDS(2));
```

Comment:
Does the changes to these test imply that scheduling is now "slower"? 

I think the commit message could use a bit more reasoning/elaboration.

Reply:
Updated the commit message, that the additional delay in timeout is introduced that so advertising retries are considered in verifying the test criteria.